vumi/persist/txredis_manager.py

```diff
@@ -154,6 +174,15 @@ def ttl(self, key):
         d.addCallback(lambda r: (None if r < 0 else r))
         return d
 
+    def persist(self, key):
+        self._send('PERSIST', key)
+        return self.getResponse()
+
+    def type(self, key):
+        d = self.get_type(key)
+        d.addCallback(lambda r: r if r is not None else 'none')
```

Comment:
Yes. There's code in txredis that turns `'none'` into `None`, but leaves all the other return values of the `TYPE` command as strings. I'll add a comment.